CONTRIBUTING.md

```diff
@@ -23,6 +23,11 @@ These things will make a PR more likely to be accepted:
 - tests for old code!
 - new code and tests follow the conventions in old code and tests
 - a good commit message (see below)
+- all code must abide by [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
+- names should abide by [What's in a name](https://talks.golang.org/2014/names.slide#1)
+- code must build on both Linux and Darwin, via plain `go build`
```

Comment:
That's a tricky one to enforce. It makes sense, but as a random person if I try to join to contribute that may make some uncertainties as I have no mac where I can test and build.

Reply:
Agreed, this is not meant to be a strict/hard requirement but rather an acknowledgment that a simple `go build` should be sufficient.